customer-config/checkout-app/cha.settings.v1.json

```diff
@@ -487,6 +487,11 @@
         },
         "layout": {
           "$ref": "#/$defs/ReceiptLayoutType"
+        },
+        "promotionBarsEnabled": {
```

Comment:
linkedPromptionsIndicatorEnabled? avoid using "Bars" as it's too specific to the UI implementation